src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1042,6 +1042,28 @@ public static bool IsDefaultExpressionContext(this SyntaxTree syntaxTree, int po
             return false;
         }
 
+        public static bool IsParamsModifierContext(
+            this SyntaxTree syntaxTree,
+            int position,
+            SyntaxToken tokenOnLeftOfPosition,
+            CancellationToken cancellationToken)
+        {
+            var token = tokenOnLeftOfPosition;
```

Comment:
move these two lines below the first check.  They're not needed until the second check.